developer/contributing/pull-requests.rst

```diff
@@ -9,15 +9,37 @@ When creating a pull request:
   example: ``feature/what-my-pr-does``. Note that dashes should be used
   instead of spaces (not underscores).
 * Add your feature/bugfix/enhancement.
+* Do atomic and logically separate commits (use the power of ``git rebase`` to
+  have a clean and logical history).
+* Write good commit messages (see the tip below).
 * Write tests.
 * :doc:`test-your-code`.
 * Add a line in the format of "<type> <pr-number>
   [<affected bundle or component>] <description>" to the ``CHANGELOG.md`` file
   in the root directory.
-* In case you have some changes breaking backwards compability you also have to
+* In case you have some changes breaking backwards compatibility you also have to
   add a description to the ``UPGRADE.md`` file in the root directory.
-* Use a meaningful name for the pull request.
+* Use a meaningful name for the pull request (see the tip below).
 * Create the pull request as soon as possible.
+* In addition to this "code" pull request, you must also send a pull request to
+  the `documentation repository`_ to update the documentation when appropriate.
+
+.. tip::
+
+    A good commit message is composed of a summary (the first line),
+    optionally followed by a blank line and a more detailed description. The
+    summary should start with the Component you are working on in square
+    brackets (``[MediaBundle]``, ``[ContentBundle]``, ...). Use a
```

Comment:
I don't [see this happening](https://github.com/sulu-io/sulu/pulls). Is this rule up to date?